include/readout/concepts/LatencyBufferConcept.hpp

```diff
@@ -29,34 +29,25 @@ class LatencyBufferConcept
   LatencyBufferConcept(LatencyBufferConcept&&) = delete;            ///< LatencyBufferConcept is not move-constructible
   LatencyBufferConcept& operator=(LatencyBufferConcept&&) = delete; ///< LatencyBufferConcept is not move-assignable
 
-  virtual void conf(const nlohmann::json& cfg) = 0;
+  virtual void resize(size_t) = 0;
```

Comment:
I would prefer written argument names in signatures. (Useful for doxy and readability. This is a bad example, as it's pretty explicit what we are doing. But still. `resize(size_t capacity)`